# test the module raise an Excepetion

import testMyFunction as tm

print(tm.withdraw(20))
print("all good")